business_service.go

```diff
@@ -76,55 +87,103 @@ func (c *Client) ListBusinessServices(o ListBusinessServiceOptions) (*ListBusine
 	}
 
 	// Make call to get all pages associated with the base endpoint.
-	if err := c.pagedGet(context.TODO(), "/business_services"+queryParms.Encode(), responseHandler); err != nil {
+	if err := c.pagedGet(ctx, "/business_services"+queryParms.Encode(), responseHandler); err != nil {
 		return nil, err
 	}
-	businessServiceResponse.BusinessServices = businessServices
 
-	return businessServiceResponse, nil
+	return businessServices, nil
 }
 
-// CreateBusinessService creates a new business service.
+// CreateBusinessService creates a new business service. It's recommended to use
+// CreateBusinessServiceWithContext instead
 func (c *Client) CreateBusinessService(b *BusinessService) (*BusinessService, *http.Response, error) {
-	data := make(map[string]*BusinessService)
-	data["business_service"] = b
-	resp, err := c.post(context.TODO(), "/business_services", data, nil)
+	return c.createBusinessServiceWithContext(context.Background(), b)
+}
+
+// CreateBusinessServiceWithContext creates a new business service.
+func (c *Client) CreateBusinessServiceWithContext(ctx context.Context, b *BusinessService) (*BusinessService, error) {
```

Comment:
@stmcallister similar to #293, I'm curious of your thoughts around removing the `*http.Response` from these methods.

Reply:
As mentioned in #293, as long as we can get the same information from somewhere--in this case, from the `APIError` object--we should be okay with this change.